src/Web/Slack/Experimental/Blocks/Types.hs

```diff
@@ -295,6 +391,9 @@ instance ToJSON SlackBlock where
       , "block_id" .=? mBlockId
       , "elements" .=! as
       ]
+  -- FIXME(jadel): should this be an error? Slack doesn't accept these
```

Comment:
Hmmm. Thanks Slack. This sucks. 

Ideally we want this to not compile. Question is how much does it matter?

Solutions could be... 
- Different types for `SlackBlockIn` and `SlackBlockOut` (I don't like those names)
- Something fancy with type classes or GADTs. 

If you need the richtext now, then coming up with the ideal solution isn't worth it yet. 

I might make the output something like `"error: cannot encode RichText"` so the ugly error will be visible.

Reply:
Agreed that this should be made to not compile. It's in Experimental so I'm going to call it a "fix it later" for now.